packages/semaphore-identity-pcd/src/SemaphoreIdentityPCD.ts

```diff
@@ -1,15 +1,23 @@
-import { DisplayOptions, PCD, PCDPackage, SerializedPCD } from "@pcd/pcd-types";
-import { requireDefinedParameter } from "@pcd/util";
-import { Identity } from "@semaphore-protocol/identity";
 import JSONBig from "json-bigint";
 import { v4 as uuid } from "uuid";
+
+import {
+  DisplayOptions,
+  IdentityArgument,
+  PCD,
+  PCDPackage,
+  SerializedPCD
+} from "@pcd/pcd-types";
+import { requireDefinedParameter } from "@pcd/util";
+import { Identity } from "@semaphore-protocol/identity";
+
 import { SemaphoreIdentityCardBody } from "./CardBody";
 
 export const SemaphoreIdentityPCDTypeName = "semaphore-identity-pcd";
 
-export interface SemaphoreIdentityPCDArgs {
-  identity: Identity;
-}
+export type SemaphoreIdentityPCDArgs = {
+  identity: IdentityArgument;
+};
```

Comment:
@artwyman related to our discussion about PCD args vs. just a prove function that lets you pass in the right stuff without the args data structure.